### PR TITLE
bump synthetix on andromeda for rewardmanager fix

### PR DESCRIPTION
**Pull request by dbeal-eth**

### CODE REVIEW

**Review by noisekit — APPROVED** [review]
Looks good to me.

File: omnibus-base-mainnet-andromeda.toml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 name = "synthetix-omnibus"
-version = "26"
+version = "27"
```
Comment (noisekit) [inline omnibus-base-mainnet-andromeda.toml:2]: Just checking if we have done testnet already?

Comment (noisekit) [reply omnibus-base-mainnet-andromeda.toml:2]: If not, then lets do base sepolia first

Comment (dbeal-eth) [reply omnibus-base-mainnet-andromeda.toml:2]: That's right we haven't

**Review by noisekit — APPROVED** [review]

**Pull request merged**